src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -116,6 +117,13 @@ public partial class HubConnection
         /// </summary>
         public TimeSpan HandshakeTimeout { get; set; } = DefaultHandshakeTimeout;
 
+        /// <summary>
+        /// Gets the connection's current Id. This value will be cleared when the connection is stopped and will have a new value every time the connection is (re)established.
+        /// This value will be null if the negotiation step is skipped via HttpConnectionOptions or if the WebSockets transport is explicitly specified because the
```

Comment:
This is incorrect. Negotiate is only skipped when Websockets **AND** `SkipNegotiate` are set.

Reply:
I think SkipNegotiate is only possible when Websockets is set. Could simplify the comment to just meantion it.